Task Management/assets/scss/globals/_mixins.scss

```diff
@@ -0,0 +1,31 @@
+@mixin flex-dir($direction){
+    flex-direction: $direction;
+}
+
+@mixin align-item($aligning){
+    align-items: $aligning;
+}
+
+@mixin text-align($align){
+    text-align: $align;
+}
```

Comment:
"Text-align, align-items, and flex-direction do not need to use mixins; they can be defined as classes since they only require one line and do not need a mixin."